great_expectations/validator/metric_configuration.py

```diff
@@ -8,7 +8,20 @@
 from great_expectations.core.util import convert_to_json_serializable
 
 
+@public_api
 class MetricConfiguration:
+    """The MetricConfiguration class provides a interface for configuring Metrics, including domain information,
```

Comment:
The first line needs to be short, more description can follow after a line break.